lib/sync/index.js

```diff
@@ -136,49 +158,45 @@ function start(cb) {
       return callback();
     },
     function startSyncScheduler(callback) {
-      syncScheduler = new SyncScheduler();
+      syncScheduler = new SyncScheduler(redisClient);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "var" keyword to this declaration of "syncScheduler". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2703)